src/components/TodoList.jsx

```diff
@@ -64,21 +65,18 @@ export const TodoWrapper = () => {
           <TodoForm addTodo={addTodo} />
           <DndContext sensors={sensors} collisionDetection={closestCenter} onDragEnd={handleDragEnd}>
               <SortableContext items={todos.map((todo) => todo.id)} strategy={verticalListSortingStrategy}>
-                  {todos.map((todo, index) => (
-                      todo.isEditing ? (
-                          <EditTodoForm key={todo.id} editTodo={editTask} task={todo}/> 
-                      ) : (
-                          <Todo 
-                              key={todo.id}
-                              task={todo}
-                              toggleComplete={toggleComplete}
-                              deleteTodo={deleteTodo} 
-                              editTodo={editTodo} 
-                          />
-                      )
```

Comment:
minor suggestion. 

From the TODOList point of view, here, im creating a <Todo /> component, if you check the properites are common , that's because the fact that it's on "edit" mode or in "regular" mode is defined by the `<Todo />` component